WcaOnRails/lib/results_validators/persons_validator.rb

```diff
@@ -19,6 +19,8 @@ class PersonsValidator < GenericValidator
     NON_MATCHING_NAME_WARNING = "The name '%{name}' provided for %{wca_id} does not match the current record in the WCA database ('%{expected_name}'). "
     NON_MATCHING_COUNTRY_WARNING = "The country '%{country}' provided for %{name} (%{wca_id}) does not match the current record in the WCA database ('%{expected_country}')."\
     " If this is an error, fix it. Otherwise, leave a comment to the WRT about it."
+    MULTIPLE_NEWCOMERS_WITH_SAME_NAME_ERROR = "There are multiple new competitors with the exact same name: %{name}. Due to a bug in the code, we cannot upload the Results. Please contact the Results Team with your JSON (free of all other errors) to help upload the results."
+    WRONG_PARENTHESIS_TYPE_ERROR = "The parenthesis character in '%{name}', please replace it with a space and regular parenthesis."
```

Comment:
The sentence looks incomplete. Proposal: "The parenthesis character used in '%{name}' is an irregular character, please replace it with a regular parenthesis '(' or ')' and with appropriate spacing."

I specifically removed "space and" because this does not apply to the closing parenthesis.

Reply:
Updated!